cmd/clusterctl/client/config/providers_client.go

```diff
@@ -98,13 +98,19 @@ const (
 // IPAM providers.
 const (
 	InClusterIPAMProviderName = "in-cluster"
+	NutanixIPAMProviderName   = "nutanix"
 )
 
 // Add-on providers.
 const (
 	HelmAddonProviderName = "helm"
 )
 
+// Runtime extensions providers.
+const (
+	NutanixRuntimeExtensionsProviderName = "nutanix"
```

Comment:
```suggestion
	NutanixRuntimeExtensionsProviderName = "nutanix-cloud-native-nutanix"
```

To follow the conventions at https://main.cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#adding-a-provider-to-clusterctl